tests/MyGame/Example/Monster.cs

```diff
@@ -129,6 +129,47 @@ public sealed class Monster : Table {
     return new Offset<Monster>(o);
   }
   public static void FinishMonsterBuffer(FlatBufferBuilder builder, Offset<Monster> offset) { builder.Finish(offset.Value, "MONS"); }
+  
+  public int KeyCompareLessThan(Monster o) { return Name.CompareTo(o.Name); }
+  public int KeyCompareWithValue(string val) { return Name.CompareTo(val); }
+
+  public static Monster[] CreateMyTableVector(Offset<Monster>[] offsets, ByteBuffer bb) {
```

Comment:
see if you can move parts of these larger functions into a standard helper function, rather than generating code for all of it.